src/aws-type-generator/src/convert.ts

```diff
@@ -234,9 +234,14 @@ function visitSchema(context: Context, factory: TypeFactory, record: SchemaRecor
             parts.shift()
         }
 
-        const definition = record.schema.definitions?.[parts[0]];
+        let definition = record.schema.definitions?.[parts[0]];
         if (!definition) {
-            throw new Error(`could not find definition ${schema.$ref} in ${context}`)
+            // Note: Definitions are typically the only schema with circular references 
```

Comment:
There is a new cloud control API that has a properties reference in definitions. We only have definitions set up with circular references (i.e. a definition referencing another definition) so we add the edge case here